Code/Tools/SceneAPI/SceneUI/SceneWidgets/ManifestWidgetPage.cpp

```diff
@@ -100,6 +102,20 @@ namespace AZ
                 return true;
             }
 
+            void ManifestWidgetPage::UpdateAddButtonStatus()
+            {
+                if (m_objects.size() >= m_capSize)
+                {
+                    ui->m_addButton->setToolTip(tr("This group container reached its cap of %1 entries.").arg(m_capSize));
```

Comment:
nit: "Maximum number of entries reached. Group containers can contain up to %1 entries."

Reply:
I removed the term "group container", I think that's a code only concept and won't make sense to content creators.